tests/integration_tests/test_grpc_only.py

```diff
@@ -45,6 +45,7 @@ def grpc_eth_call(port: int, args: dict, chain_id=None, proposer_address=None):
     ).json()
 
 
+@pytest.mark.skip(reason="undeterministic test - https://github.com/evmos/ethermint/issues/1530")
```

Comment:
break line

Reply:
Either break line or we could add a flake8 config file as we have recently decided on 120 characters max line length for the [proto files](https://github.com/evmos/ethermint/blob/main/.protolint.yml#L74)?

Reply:
@MalteHerrmann yes, let's add that